src/main/java/io/logz/guice/jersey/JerseyModule.java

```diff
@@ -5,23 +5,38 @@
 import com.google.inject.Provider;
 import com.google.inject.servlet.ServletModule;
 import io.logz.guice.jersey.configuration.JerseyConfiguration;
+import io.logz.guice.jersey.configuration.JettyConfiguration;
 
 import java.util.Objects;
 
 public class JerseyModule extends AbstractModule {
 
     private final JerseyConfiguration jerseyConfiguration;
+    private JettyConfiguration jettyConfiguration;
 
     public JerseyModule(JerseyConfiguration jerseyConfiguration) {
         this.jerseyConfiguration = Objects.requireNonNull(jerseyConfiguration);
     }
 
+    public JerseyModule(JerseyConfiguration jerseyConfiguration, JettyConfiguration jettyConfiguration) {
```

Comment:
1. import static requireNonNull 
2. don't use 2 different implementations to the constructor and don't allow jettyConfiguration to be null. either make it required or add an overload constructor that passes default jetty config.
that way we don't need to add null checks in the rest of the flow and we won't have 2 implementation for the JerseyServer constructor as well

Reply:
@asafalima Regarding #2, I agree that it will make the code nicer. Just please note it means to hard code what we assume Jetty’s defaults are (as they are used in a specific jetty version we currently choose to support). In the original PR code the jetty server itself sets its defaults so we’re agnostic to its version or default configuration. Let me know which one you prefer.

Reply:
We're setting it to Jetty's defaults in that point in time in order to avoid changing previous behavior.
it doesn't matter if they will change it or not, the user will be able to configure the queue he wants by himself.